### DISCUSSION TITLE
[FEATURE] Valine visitors not work on index.html

### DISCUSSION THREAD

**Discussion by Rhinocros** [HEIGE-PCloud/DoIt#193] [Q&A] [answered]
我复制了
```
 <span id="{{ .RelPermalink }}" class="leancloud_visitors" data-flag-title="{{ .Title }}">
                    <i class="far fa-eye fa-fw"></i>&nbsp;<span class=leancloud-visitors-count></span>&nbsp;{{ T
                    "views" }}
                </span>
```
到 summary.html 和 index.html ，但 `hugo server -e production` 后并没能把读数回填，正事生成后也不行， single.html 页面正常，请问怎么能在首页上显示已读次数

**Comment by HEIGE-PCloud** [comment, marked as answer]
Valine 的访客计数是基于 Valine 评论区的。然而首页上并没有放置 Valine 评论区，相关的计数功能自然也不会显示。

在 `index.html` 的页脚添加 `{{- partial "comment.html" . -}}` 在首页上加入评论区以启用 Valine 的访客计数。
